odl_video/settings.py

```diff
@@ -211,6 +298,127 @@ def get_var(name, default):
+MAILGUN_FROM_EMAIL = get_var('MAILGUN_FROM_EMAIL', 'no-reply@example.com')
+MAILGUN_BCC_TO_EMAIL = get_var('MAILGUN_BCC_TO_EMAIL', 'no-reply@example.com')
+
+
+# e-mail configurable admins
+ADMIN_EMAIL = get_var('ODL_VIDEO_ADMIN_EMAIL', '')
+if ADMIN_EMAIL != '':
+    ADMINS = (('Admins', ADMIN_EMAIL),)
+else:
+    ADMINS = ()
+
+# Logging configuration
+LOG_LEVEL = get_var('ODL_VIDEO_LOG_LEVEL', 'INFO')
+DJANGO_LOG_LEVEL = get_var('DJANGO_LOG_LEVEL', 'INFO')
+
+# For logging to a remote syslog host
+LOG_HOST = get_var('ODL_VIDEO_LOG_HOST', 'localhost')
+LOG_HOST_PORT = get_var('ODL_VIDEO_LOG_HOST_PORT', 514)
+
+HOSTNAME = platform.node().split('.')[0]
+DEFAULT_LOG_STANZA = {
+    'handlers': ['console', 'syslog'],
+    'level': LOG_LEVEL,
+}
+
+# nplusone profiler logger configuration
+NPLUSONE_LOGGER = logging.getLogger('nplusone')
+NPLUSONE_LOG_LEVEL = logging.ERROR
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': True,
+    'filters': {
+        'require_debug_false': {
+            '()': 'django.utils.log.RequireDebugFalse',
+        }
+    },
+    'formatters': {
+        'verbose': {
+            'format': (
+                '[%(asctime)s] %(levelname)s %(process)d [%(name)s] '
+                '%(filename)s:%(lineno)d - '
+                '[{hostname}] - %(message)s'
+            ).format(hostname=HOSTNAME),
+            'datefmt': '%Y-%m-%d %H:%M:%S'
+        }
+    },
+    'handlers': {
+        'console': {
+            'level': 'DEBUG',
+            'class': 'logging.StreamHandler',
+            'formatter': 'verbose'
+        },
+        'syslog': {
+            'level': LOG_LEVEL,
+            'class': 'logging.handlers.SysLogHandler',
+            'facility': 'local7',
+            'formatter': 'verbose',
+            'address': (LOG_HOST, LOG_HOST_PORT)
+        },
+        'mail_admins': {
+            'level': 'ERROR',
+            'filters': ['require_debug_false'],
+            'class': 'django.utils.log.AdminEmailHandler'
+        },
+        'sentry': {
+            'level': 'ERROR',
+            'class': 'raven.contrib.django.raven_compat.handlers.SentryHandler',
+            'formatter': 'verbose'
+        },
+    },
+    'loggers': {
+        'root': DEFAULT_LOG_STANZA,
+        'odl_video': DEFAULT_LOG_STANZA,
+        'django': {
+            'propagate': True,
+            'level': DJANGO_LOG_LEVEL,
+            'handlers': ['console', 'syslog'],
+        },
+        'django.request': {
+            'handlers': ['mail_admins'],
+            'level': DJANGO_LOG_LEVEL,
+            'propagate': True,
+        },
+        'urllib3': {
+            'level': 'INFO',
+        },
+        'raven': {
+            'level': 'DEBUG',
+            'handlers': []
+        },
+        'nplusone': {
+            'handlers': ['console'],
+            'level': 'ERROR',
+        }
+    },
+}
+
+# AWS
+AWS_ACCESS_KEY_ID = os.environ.get('AWS_ACCESS_KEY_ID', '')
```

Comment:
Some of these are also defined above, and they use `get_var` above instead of `os.environ` here. There are also S3 settings defined elsewhere. Can you move all the AWS settings into one place?